app/models/claim.rb

```diff
@@ -91,19 +91,24 @@ def data_for_calculation
       disbursements: disbursements_for_calculation,
       letters_and_calls: letters_and_calls_for_calculation,
       youth_court: data['youth_court'] == 'yes',
-      # TODO: CRM457-2306: Amend these as the fields will exist
-      claimed_youth_court_fee_included: data['include_youth_court_fee'],
+      claimed_youth_court_fee_included: youth_court_fee_allowed,
       plea_category: data['plea_category'],
-      # TODO: CRM457-2306: Remove these as the fields will exist
       # :nocov:
-      assessed_youth_court_fee_included: data['youth_court_fee_adjustment_comment']&.present?,
+      assessed_youth_court_fee_included: data['include_youth_court_fee'],
       # :nocov:
     }
   end
-  # rubocop:enable Metrics/AbcSize
 
   private
 
+  def youth_court_fee_allowed
```

Comment:
I think this should be called `youth_court_fee_claimed`

Reply:
I also preferred your data.fetch call

Reply:
will update on both counts